tests/cli/test_compose_sanity.sh

```diff
@@ -22,6 +22,30 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        #wait for the build to properly start
+        sleep 20
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose info"
+        if [ -n "$UUID" ]; then
+            rlRun -t -c "$CLI compose info $UUID | egrep 'RUNNING|WAITING'"
+        else
+            rlFail "Compose UUID is empty!"
+        fi
+    rlPhaseEnd
```

Comment:
this section is not needed. Instead it should be moved above and executed within a loop until the test detects a RUNNING status. Should look like `wait_for_compose` function (or even reuse the function by passing parameter RUNNING)

Reply:
Ok I took the `wait_for_compose` function and reused it for detecting the running compose. I am just not sure about the sleep duration and sometimes it didn't even sleep for me because the compose was already running and that then changes the cancel behavior. We'll need Brian's input on this.

Reply:
The above looks good in terms of how we wait for the compose to start running IMO.